ospd/scan.py

```diff
@@ -397,6 +397,11 @@ def get_vts(self, scan_id: str) -> Dict:
 
         return self.scans_table[scan_id]['vts']
 
+    def release_vts_list(self, scan_id: str) -> None:
+        """ Release the memory used for the vts list. """
+
+        self.scans_table[scan_id].pop('vts')
```

Comment:
i think pop may raise a keyerror. i don't think this can happen in our case but maybe it is safer to use
```suggestion
        scan_data = self.scans_table.get(scan_id)
        if scan_data and 'vts' in scan_data:
            delete scan_data['vts']
```